content/blogs/Fuzzing-ReactOS.md

```diff
@@ -0,0 +1,43 @@
+---
+title:       "Syzkaller - Fuzzing ReactOS in 2020"
+author:      "Freakston"
+date:        2020-08-04
+tags:        [ "gsoc", "gsoc-2020" ]
+---
+
+Hello everyone. In this blog post, I will be talking about the steps to set up Syzkaller for ReactOS locally and start fuzzing. 
+If you have been following the blog posts on the ReactOS website then you would have noticed that this is my Google Summer of Code 2020 Project. 
+Now that Coding period 2 has officially come to an end and the work is nearing completion, this post will help others in setting up the fuzzer get started with fuzzing. 
+
+If you haven't checked out the posts regarding the progress, here they are -
+
+[PORTING SYZKALLER TO REACTOS](https://reactos.org/blogs/porting-syzkaller-to-reactos-gsoc-2020/)
+
+[PORTING SYZKALLER TO REACTOS (EVALUATION-1)](https://reactos.org/blogs/porting-syzkaller-to-reactos-1-gsoc2020/)
+
+# Environment setup
+
+1. **Virtual box or QEMU**:
+Most of the Virtualization software supported by ReactOS works properly. If using QEMU, it 's recommended to use with KVM enabled for better performance. (**Tested on Virtual box and QEMU.**)
+
+2. **ReactOS**:
+Running Syzkaller needs certain NT6 API's, which aren't exported in the current master. 
+To get a copy of reactOS for Syzkaller head to [PR-2930](https://github.com/reactos/reactos/pull/2930) and get the MSVC-i386 build from checks. 
+This is automatically built by Github Actions.
+In case you want to build it on your own then you can clone my [fork](https://github.com/Freakston/reactos/tree/syzkaller) of reactOS and build it using MSVC.
```

Comment:
```suggestion
To get a copy of ReactOS for Syzkaller head to [PR #2930](https://github.com/reactos/reactos/pull/2930) and get the MSVC-i386 build from checks. 
This is automatically built by GitHub Actions.
In case you want to build it on your own then you can clone my [fork](https://github.com/Freakston/reactos/tree/syzkaller) of ReactOS and build it using MSVC.
```